thorlcr/slave/slave.hpp

```diff
@@ -87,6 +88,8 @@ interface IThorDataLink : extends IRowStream
     virtual CActivityBase *queryFromActivity() = 0; // activity that has this as an output
     virtual void dataLinkSerialize(MemoryBuffer &mb)=0;
     virtual unsigned __int64 queryTotalCycles() const=0;
+    virtual void debugRequest(MemoryBuffer &mb) = 0;
+    virtual void setInputSplitterOutputBase(CSplitterOutputBase *_input, unsigned _id=0) = 0;
```

Comment:
This doesn't look right.. a method to do with Splitter activity being a virtual member of interface IThorDataLink.